qa/src/test/resources/features/datastore/Datastore.feature

```diff
@@ -331,6 +331,270 @@ Feature: Datastore tests
     And Client "test-client-2" last published on a channel in the list "ChannelList" on "03/07/2017T09:00:20"
     And All indices are deleted
 
+  Scenario: Account wide metrics check
+    Check the correctness of the metric info data stored by retrieving the metrics information by account.
+
+    Given I login as user with name "kapua-sys" and password "kapua-password"
+    And Account for "kapua-sys"
+    And The device "test-device-1"
+    And I set the database to device timestamp indexing
+    Then I prepare a number of messages with the following details and remember the list as "TestMessages"
+      |clientId      |topic            |
+      |test-client-1 |test_topic/1/2/3 |
+      |test-client-2 |test_topic/1/2/3 |
+    And I set the following metrics to the message 0 from the list "TestMessages"
+      |metric        |type      |value      |
+      |tst-metric-1  |double    |123        |
+      |tst-metric-2  |int       |123        |
+    And I set the following metrics to the message 1 from the list "TestMessages"
+      |metric        |type      |value      |
+      |tst-metric-3  |string    |123        |
+      |tst-metric-4  |bool      |true       |
+    Then I store the messages from list "TestMessages" and remember the IDs as "StoredMessageIDs"
+    And I refresh all database indices
+    When I query for the current account metrics and store them as "AccountMetrics"
+    Then There are exactly 4 metrics in the list "AccountMetrics"
+    And The metric info items "AccountMetrics" match the prepared messages in "TestMessages"
+    And All indices are deleted
+
+  Scenario: MetricsInfo client ID and topic data based on the client id
```

Comment:
the metrics are different (see https://github.com/eclipse/kapua/blob/develop/service/datastore/internal/src/test/java/org/eclipse/kapua/service/datastore/internal/MessageStoreServiceTest.java#L1151)
TestMessage2 should have the same metric names of the TestMessages[1] but the values of the TestMessages[0]
The idea under this test is to mix the metrics type using the same metric name. So the client-1 should publish a message with the same metric names of one already published by the client-0, but with different value type.